### ISSUE TITLE
Learn more link on home page points to placeholder page

### ISSUE THREAD

**Issue by kravciak** [rancher/dashboard#2925]
Getting Started -> `Learn More` on home page points to empty (placeholder) page with content:

```
Home > Getting Started

Gettting Started
Welcome to Rancher Dashboard.
```

Tested on master-head (909237a)

**Comment by nwmac** [comment]
This will be fixed as part of rancher/dashboard#2735

**Comment by nwmac** [comment]
Close as duplicate
